tests/compose-tests/test-installroot.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+dn=$(cd $(dirname $0) && pwd)
+. ${dn}/libcomposetest.sh
+
+prepare_compose_test "installroot"
+compose_base_argv="--repo=${repobuild} --cache-only --cachedir=${test_compose_datadir}/cache"
+instroot_tmp=$(mktemp -d /var/tmp/rpm-ostree-instroot.XXXXXX)
+rpm-ostree compose install ${compose_base_argv} ${treefile} ${instroot_tmp}
+instroot=${instroot_tmp}/rootfs
+assert_not_has_dir ${instroot}/usr/lib/ostree-boot
+assert_not_has_dir ${instroot}/etc
+test -L ${instroot}/home
+assert_has_dir ${instroot}/usr/etc
+
+# Clone the root - we'll test direct commit, as well as postprocess with
+# and without treefile.
+mv ${instroot}{,-postprocess}
+cp -al ${instroot}{-postprocess,-directcommit}
+cp -al ${instroot}{-postprocess,-postprocess-treefile}
+
+integrationconf=usr/lib/tmpfiles.d/rpm-ostree-0-integration.conf
+
+assert_not_has_file ${instroot}-postprocess/${integrationconf}
+rpm-ostree compose postprocess  ${instroot}-postprocess
+rpm-ostree compose postprocess ${instroot}-postprocess-treefile
```

Comment:
Did you mean to pass the treefile here? (And maybe also a check of some kind afterwards to make sure treefile info was applied?)

Reply:
Yes!  Also a very good catch, thanks.  Both fixed ⬇️